core/src/main/java/org/apache/calcite/adapter/enumerable/RexImpTable.java

```diff
@@ -844,6 +845,7 @@ Builder populate2() {
       defineMethod(ARRAYS_ZIP, BuiltInMethod.ARRAYS_ZIP.method, NullPolicy.ANY);
       defineMethod(EXISTS, BuiltInMethod.EXISTS.method, NullPolicy.ANY);
       defineMethod(MAP_CONCAT, BuiltInMethod.MAP_CONCAT.method, NullPolicy.ANY);
+      defineMethod(MAP_CONTAINS_KEYS, BuiltInMethod.MAP_CONTAINS_KEYS.method, NullPolicy.ANY);
```

Comment:
`MAP_CONTAINS_KEYS` -> `MAP_CONTAINS_KEY`?

Reply:
done, thank you